core/js/pokemon.maps.js

```diff
@@ -323,51 +352,22 @@ function addPokemonMarker(pokemon,pokeimg_suffix, locale) {
 		anchor: new google.maps.Point(16, 16),
 		labelOrigin : new google.maps.Point(16, 36)
 	};
-	var encounter = false;
+	var encountered = false;
 	var ivPercent = 100;
 	if (pokemon.individual_attack !== null) {
-		encounter = true;
+		encountered = true;
 		ivPercent = ((100/45)*(parseInt(pokemon.individual_attack)+parseInt(pokemon.individual_defense)+parseInt(pokemon.individual_stamina))).toFixed(2);
 	}
 	var marker = new google.maps.Marker({
 		position: {lat: parseFloat(pokemon.latitude), lng:parseFloat(pokemon.longitude)},
-		map: map,
 		icon: image,
-		encounterId: pokemon.encounter_id,
 		ivPercent: ivPercent
 	});
-	if (encounter) {
-		marker.setLabel({
-			color:getIvColor(ivPercent),
-			text:ivPercent+"%"
-		});
+	if (encountered) {		
+		marker.setLabel(getMarkerLabel(ivPercent));
 	}
-	var contentString = '<div>'+
-			'<h4> '+pokemon.name+' #'+pokemon.pokemon_id+ (encounter?' IV: '+ivPercent+'% ':'')+'</h4>'+
-			'<div id="bodyContent">'+
-				'<p class="disappear_time_display text-center">'+pokemon.disappear_time_real+'<span class="disappear_time_display_timeleft"></span></p>';
-	if (encounter) {
-		contentString +=
-				'<p></p>'+
-				'<div class="progress" style="height: 6px; width: 120px; margin-bottom: 10px; margin-top: 2px; margin-left: auto; margin-right: auto">'+
-					'<div title="'+locale.ivAttack+': '+pokemon.individual_attack+'" class="progress-bar progress-bar-danger" role="progressbar" aria-valuenow="'+pokemon.individual_attack+'" aria-valuemin="0" aria-valuemax="45" style="width: '+(((100/15)*pokemon.individual_attack)/3)+'%">'+
-						'<span class="sr-only">'+locale.ivAttack+': '+pokemon.individual_attack+'</span>'+
-					'</div>'+
-					'<div title="'+locale.ivDefense+': '+pokemon.individual_defense+'" class="progress-bar progress-bar-info" role="progressbar" aria-valuenow="'+pokemon.individual_defense+'" aria-valuemin="0" aria-valuemax="45" style="width: '+(((100/15)*pokemon.individual_defense)/3)+'%">'+
-						'<span class="sr-only">'+locale.ivDefense+': '+pokemon.individual_defense+'</span>'+
-					'</div>'+
-					'<div title="'+locale.ivStamina+': '+pokemon.individual_stamina+'" class="progress-bar progress-bar-success" role="progressbar" aria-valuenow="'+pokemon.individual_stamina+'" aria-valuemin="0" aria-valuemax="45" style="width: '+(((100/15)*pokemon.individual_stamina)/3)+'%">'+
-						'<span class="sr-only">'+locale.ivStamina+': '+pokemon.individual_stamina+'</span>'+
-					'</div>'+
-				'</div>'+
-				'<p class="text-center">('+pokemon.individual_attack+"/"+pokemon.individual_defense+"/"+pokemon.individual_stamina+')</p>'+
-				'<p class="text-center">'+pokemon.quick_move+"/"+pokemon.charge_move+'</p>';
-		}
-		contentString +='</div>';
 
-	var infoWindow = new google.maps.InfoWindow({
-		content: contentString
-	});
+	var infoWindow = new google.maps.InfoWindow(getPokemonContent(pokemon, locale, encountered, ivPercent));	
```

Comment:
Could you please add in InfoWindow options:
"disableAutoPan: true" ?

This will stop the map jumping around if you touch an image and infowindow appears.
We need this also in nests.maps.js